react/features/unsupported-browser/components/UnsupportedMobileBrowser.js

```diff
@@ -82,7 +83,11 @@ class UnsupportedMobileBrowser extends Component<*, *> {
         // appears to be a link with an app-specific scheme, not a Universal
         // Link.
         const appScheme = interfaceConfig.MOBILE_APP_SCHEME || 'org.jitsi.meet';
-        const joinURL = `${appScheme}:${window.location.href}`;
+
+        // Replace the protocol part with the app scheme.
+        const joinURL
+            = window.location.href.replace(
+                new RegExp(`^${URI_PROTOCOL_PATTERN}`), `${appScheme}:`);
```

Comment:
Do you think you need the 'gi' options for this regex? I think href is always lowercase so you probably don't need it.

Reply:
I did not want gi there, because I only want to replace first occurrence of the protocol part. Does it make sense (not sure if I understand the g option correctly) ?